.github/workflows/openvex.yaml

```diff
@@ -0,0 +1,35 @@
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        type: string
+        required: true
+        description: "GitHub release tag"
+
+permissions: {}
+
+jobs:
+  vexctl:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Set tag name
+        shell: bash
+        run: |
+          TAG=${{ github.event.inputs.tag }}
+          echo "TAG=$TAG" >> "$GITHUB_ENV"
+      - uses: openvex/generate-vex@31b415924ea0d72ed5f2640f1dee59dea6c2770b
+        name: Run vexctl
+        with:
+          product: pkg:generic/karpenter@${{ env.RELEASE_VERSION }}
```

Comment:
Please compare the `product` field with this [kueue/pull/1475/files#diff-R26](https://github.com/kubernetes-sigs/kueue/pull/1475/files#diff-c2573174fef0722ce49b7b168ac16b86d880224801aa6a45af9489ed4031648cR26).

I believe it needs a bit of change; maybe remove `RELEASE_VERSION` and instead use `TAG`.